apps/explorer/src/app/components/blocks/block-data.tsx

```diff
@@ -16,7 +16,7 @@ export const BlockData = ({ block, className }: BlockProps) => {
       aria-label={`Data for block ${block.header?.height}`}
       className={className}
     >
-      <TableRow data-testid="block-row" modifier="background">
+      <TableRow data-testid="block-list" modifier="background">
```

Comment:
I feel like the testid naming is a bit off here. This element should surely should be labelled as 'block-row'. Likewise in blocks-data.tsx block-table is being applied to a list item element? Seems to be the wrong way around to me?

Reply:
Yeah I agree with this. Done